phenol-annotations/src/test/java/org/monarchinitiative/phenol/annotations/formats/hpo/category/HpoCategoryMapTest.java

```diff
@@ -15,6 +15,7 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@Deprecated
```

Comment:
I think we don't have to deprecate tests. Removing the tested class will result in compile error which will suggest us to trash it.